src/vs/workbench/contrib/tasks/common/taskConfiguration.ts

```diff
@@ -739,17 +739,15 @@ export interface IParseContext {
 }
 
 
-namespace ShellConfiguration {
```

Comment:
I restricted the mangling to top level exports so this can probably be reverted for now